geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -211,6 +211,53 @@ abstract class TiledRasterRDD[K: SpatialComponent: AvroRecordCodec: JsonFormat:
     withRDD(rdd.convert(CellType.fromName(newType)))
 
   protected def withRDD(result: RDD[(K, MultibandTile)]): TiledRasterRDD[_]
+
+  def singleTileLayerRDD: TileLayerRDD[K] = TileLayerRDD(
+    rdd.map({ case (k, v) => (k, v.band(0)) }),
+    rdd.metadata
+  )
```

Comment:
This is rough, I guess we only have polygonal summaries defined on single-band tiles in geotrellis at the moment.